addons/website/static/src/snippets/s_social_media/options.js

```diff
@@ -130,6 +130,7 @@ options.registry.SocialMedia = options.Class.extend({
                         anchorEl = this.$target[0].querySelector(':scope > a').cloneNode(true);
                         this._removeSocialMediaClasses(anchorEl);
                     }
+                    anchorEl.setAttribute("aria-label", _t("Other social network"));
```

Comment:
I guess this is the follow up of https://github.com/odoo/odoo/pull/140453/commits/d2e7a00562c9c61437944ed938f54b155c07417a#r1399321290

Just tested and this is what I have now:
- Add the social media snippet
- Toggle off the "twitter" button
- Add it back
- Now its arial-label is "Other social network"

...